.github/workflows/ci.yml

```diff
@@ -0,0 +1,56 @@
+# https://www.docker.com/blog/docker-v2-github-action-is-now-ga/
+# https://www.docker.com/blog/multi-arch-build-and-images-the-simple-way/
+# https://docs.github.com/en/actions/guides/publishing-docker-images
+
+name: CD - Build and push multi-arch images
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - '*'
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+
+      - name: Log in to the Container registry
+        uses: docker/login-action@f054a8b539a109f9f41c372932f1ae047eff08c9
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+#      - name: Login to DockerHub
+#        uses: docker/login-action@v1
+#        with:
+#          username: ${{ secrets.DOCKERHUB_USERNAME }}
+#          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Extract metadata (tags, labels) for Docker
+        id: meta
+        uses: docker/metadata-action@98669ae865ea3cffbcbaa878cf57c20bbf1c6c38
+        with:
+          images: |
+            ghcr.io/${{ github.repository }}
+
+      - name: Build and push
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: ./Dockerfile
+          platforms: linux/386,linux/amd64,linux/arm/v6,linux/arm/v7,linux/arm64
```

Comment:
@DRuggeri These are the platforms which were supported originally. Would be nice to integrate them to be able to run this on SBCs like RaspPi

Reply:
I see now... but is there a practical difference between that list of architectures and "linux/arm" + "linux/arm64"? The golang binary is the same (it only differentiates between "arm" and "arm64") and it's a scratch-based container so there are no other architecture-related files to be concerned with.

Have you tested on a Pi with Docker? I'm not at a spot to do that right now, but it ought to be functional as it is

Reply:
You are right! The container is running smoothly on my RockPi.
Thanks for your help!